### DISCUSSION TITLE
Android Auto metadata old?

### DISCUSSION THREAD

**Discussion by anthonysciutto** [advplyr/audiobookshelf-app#289] [Q&A]
I fear I'm missing something obvious, but how do I get Android Auto to 'refresh' its cached metadata? I launched the AndroidAuto app first then started adding proper cover art + chapters for my books and that information doesn't show up. 

In the app I see it but not when connected to my car.

Am I missing something?

**Comment by advplyr** [comment]
Fully close and re-open the app or log out from your server and log back in. That should refresh it.

**Reply by anthonysciutto** [reply]
Do chapters get displayed on AndroidAuto when listening or just Book Title + Author?

By logging out and back in I was able to get some meta-data to update but there are still cover arts not updated.

**Reply by advplyr** [reply]
Unfortunately downloaded book cover art is not yet working. I'm not sure why.

Chapters are not yet supported I haven't figured out how that will be implemented yet for m4b audiobooks.

Android auto has been a bit of a challenge but it's coming along.

**Reply by anthonysciutto** [reply]
Appreciate the response!

**Reply by Budlyte** [reply]
> Android auto has been a bit of a challenge but it's coming along.

It sure is. I connect, press Play on my console, and listen to my book while driving. Your app has come a long way and is now as easy to use as anything else.  Great work!  We all appreciate it.
